app/controllers/hosts_controller.rb

```diff
@@ -305,6 +305,8 @@ def nics
   end
 
   def ipmi_boot
+    raise Foreman::Exception.new(
```

Comment:
This might need to be in the API v2 hosts controller (or the model) too? it calls ipmi_boot without any checking.